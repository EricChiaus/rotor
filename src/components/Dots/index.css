.dots {
    width: 5px;
    background-color: black;
}

.dots:before,
.dots:after {
    display: inline-block;
    width: 5px;
    height: 5px;
    content: '';
    position: relative;
    left: 0;
    background-color: white;
}
