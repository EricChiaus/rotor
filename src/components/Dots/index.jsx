import React from 'react';
import './index.css';

const Dots = () => <div className="dots"></div>;

export default Dots;
