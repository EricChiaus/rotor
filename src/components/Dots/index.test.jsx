import { render } from '@testing-library/react';
import Dots from '.';

describe('<Dots />', () => {
    it('matches the snapshot', () => {
        const { container } = render(<Dots />);
        expect(container).toMatchSnapshot();
    });
});
