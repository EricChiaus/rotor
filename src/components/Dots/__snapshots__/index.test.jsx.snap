// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dots /> matches the snapshot 1`] = `
<div>
  <div
    class="dots"
  />
</div>
`;
